### DISCUSSION TITLE
Issue with 'Wipe Archive' feature, NetworkError

### DISCUSSION THREAD

**Discussion by novusB** [victornpb/undiscord#607] [1. Questions & Answers]
When I initially started wiping all of my discord, I had to remove servers that I don't have access to any longer, I had to remove accounts which became 'unknown'

I have about 1054 "jobs/DMs" to remove messages from, I get maybe 10-15 "jobs" in and then I receive this message every time.

Search request threw an error:	{"fileName":"","lineNumber":0,"columnNumber":1,"message":"NetworkError when attempting to fetch resource."}
CoreException	{"fileName":"","lineNumber":0,"columnNumber":1,"message":"NetworkError when attempting to fetch resource."}

**Comment by Crimsoth** [comment]
Getting the same problem.

**Comment by novusB** [comment]
One of the things I think that has worked for me, but greatly increases delete time...

https://jsoneditoronline.org/

I went to this website, pasted my index.json file into the editor.. I then copied the new json in the 'formatted' version, renamed the original index.json file to index1.json and then created a new index.json and uploaded that -  which takes it from a single line to each new channel on separate lines.

I increased my search time from 30000ms to 60000ms(1 min) and then increased my delete time to 1500ms.

It seems to be doing a much better time and it hasn't stopped one time.

**Comment by novusB** [comment]
--Update 2 hours later--

Doing the above steps has fixed the 'networkerror' error for me, the script has been deleting messages for the past 2 hours without stopping one time. I believe this fixed the issue by changing the format of the .json and increasing search and delete times.

**Comment by Crimsoth** [comment]
No thank you. I shouldn't have to download something else to make this extension work. All of this seems kinda sus.

**Reply by novusB** [reply]
You aren't downloading anything at all?

Maybe I explained incorrectly. 

https://jsoneditoronline.org/ is a website, you do not download anything. You paste your index.json file that you downloaded from discord, it takes your index.json file from a single file and formats the json. You remove all of the contents of your index.json file and then paste the new "formatted" json contents into your index.json file.

In the undiscord script, you go to 'advanced settings' change the 'search delay' to 60000ms(1 min) and then the 'delete delay' to 1500ms(1.5s)

**Reply by ghost** [reply]
@novusB hey man is it still working for you? if  it is can you show me how to get this thing working for me I would very appreciate if u help me my discord is ''realmfg''
